### ISSUE TITLE
Publish latest version

### ISSUE THREAD

**Issue by sugarshin** [producthunt/chai-enzyme#178]
HI,

Available this warning when npm installation.

```
warning "chai-enzyme@0.7.0" has unmet peer dependency "react-addons-test-utils@^0.14.0 || ^15.0.0-0".
```

Please latest version publish to npm .

**Comment by ayrton** [comment]
Related PR https://github.com/producthunt/chai-enzyme/pull/166

**Comment by ljharb** [comment]
Yes, this is intended, since it's a requirement of enzyme's.

**Comment by ayrton** [comment]
@ljharb do we have to revert #166 or is everything all good right now?

**Comment by ljharb** [comment]
I don't see why; the only way someone could run into this is if they were not explicitly depending on things enzyme requires. This is a bug fix.

**Comment by sugarshin** [comment]
I mean master branch and latest version published to npm are different.

**Comment by ljharb** [comment]
v0.7.0 was released 2 days ago; https://github.com/producthunt/chai-enzyme/commit/3081e30e7346671b458a2a3bef8c4e468b535288 is the only commit not yet released.

Another publish of v0.7.1 would be fine, sure.

**Comment by ayrton** [comment]
Released 0.7.1 just now
